synapse/api/constants.py

```diff
@@ -198,6 +198,8 @@ class EventContentFields:
     # cf https://github.com/matrix-org/matrix-doc/pull/1772
     ROOM_TYPE = "type"
 
+    ROOM_CREATOR = "creator"
```

Comment:
It might be obvious, but some comment as to where this is used would be good:

```suggestion
    # The creator of the room, as used in `m.room.create` events.
    ROOM_CREATOR = "creator"
```